vignettes/advs.Rmd

```diff
@@ -113,9 +123,18 @@ bmi_for_age <- who_bmi_for_age_boys %>%
         SEX == 2 ~ "F",
         TRUE ~ NA_character_
       ),
-      AGE = AGE * 30.4375
-    )) %>%
-  # AGEU is added in metadata, required for derive_params_growth_age()
+      # Ensure first that Age unit is "DAYS"
+      AGE = round(AGE * 30.4375)
+    ) %>%
+    # Interpolate the AGE by SEX
+    derive_interp_records(
+      by_vars = exprs(SEX),
+      parameter = "BMI"
+    ) %>%
+    # Keep patients >= 2 yrs till 20 yrs - Remove duplicates for 730 Days old which
+    # must come from WHO metadata only
+    filter(AGE >= 730.5 & AGE <= 7305)) %>%
+  # AGEU is added in metadata, required for derive_params_growth_age
```

Comment:
In the comment line 137, add the brackets for the function e.g. derive_params_growth_age()

Reply:
Cosmetic update: In the displayed data, is it possible to order the variables as following: AGE, AGEU, SEX, L, M, S, P95, Sigma ?

Reply:
sure, updated both